doonut-app-external-api/src/main/kotlin/com/doonutmate/fcm/controller/FcmController.kt

```diff
@@ -0,0 +1,43 @@
+package com.doonutmate.fcm.controller
+
+import com.doonutmate.fcm.client.dto.Notification
+import com.doonutmate.fcm.service.FcmService
+import com.doonutmate.oauth.configuration.Authorization
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.Parameter
+import io.swagger.v3.oas.annotations.tags.Tag
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@Tag(name = "FCM")
+@RequestMapping("/fcm")
+class FcmController(
+    val fcmService: FcmService,
+) {
+    @Operation(summary = "FCM 알림 전송")
+    @PostMapping("/send")
+    fun sendFcmNotification(
+        @RequestBody
+        req: Notification,
+        @RequestParam(defaultValue = "false") useNickname: Boolean,
+    ): Any {
+        return fcmService.sendNotification(req, useNickname)
+    }
```

Comment:
이건 단순 궁금증인데요, return 타입이 Any인 것과 아닌 것의 차이가 무엇인가요?

```suggestion
) {
        fcmService.sendNotification(req, useNickname)
    }
```

Reply:
결과적으로 보았을때 같습니다. `sendNotification `메서드 반환이 없어서 결국 `Unit`을 반환하기 때문입니다.

하지만 반환(return)을 명시했는데 `Unit`메서드를 리턴하는건 의도가 명확하지 않고 타입 안정성이 떨어져 보이네요

따라서, 같은 기능을 수행하지만 수정해야 할거같습니다.